src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment.service.ts

```diff
@@ -18,12 +17,13 @@ export class FileUploadAssessmentsService {
 
     constructor(private http: HttpClient) {}
 
-    saveAssessment(feedbacks: Feedback[], submissionId: number, submit = false): Observable<Result> {
-        let url = `${this.resourceUrl}/file-upload-submissions/${submissionId}/feedback`;
+    saveAssessment(feedbacks: Feedback[], submissionId: number, submit = false): Observable<EntityResponseType> {
+        const params = new HttpParams();
         if (submit) {
-            url = buildUrlWithParams(url, ['submit=true']);
+            params.set('submit', Boolean(true).toString());
```

Comment:
Does it really make sense to use `Boolean(true).toString()` ? Could we not just use `'true'` ?